packages/next/src/lib/metadata/resolve-metadata.test.ts

```diff
@@ -177,6 +177,41 @@ describe('accumulateMetadata', () => {
       })
     })
 
+    it('should fill only the existing props from openGraph to twitter', async () => {
```

Comment:
this test seems to pass on the current canary, I wonder if there's another path that needs to be looked at

Reply:
Update the test case in [d0b6fa2](https://github.com/vercel/next.js/pull/52196/commits/d0b6fa21a369bd7d0bb3b03500dd7af6691b7c34), turns out it fails when there's static metadata files existing will trigger it. Nice catch

Reply:
nice! looks good. should we update the type of `ResolvedTwitterSummary` to mark `title` as optional?